python/README.md

```diff
@@ -1,3 +1,18 @@
+#
+#  Copyright 2017 TWO SIGMA OPEN SOURCE, LLC
```

Comment:
Sorry! I meant to have the Copyright clause in proper markup multi line comments `<!-- ... -->`.
Fixed now.